src/components/input/Input.tsx

```diff
@@ -28,6 +28,8 @@ export interface IInputOwnProps {
     validateOnMount?: boolean;
     autoFocus?: boolean;
     disabledTooltip?: string;
+    minimum?: string;
+    maximum?: string;

```

Comment:
Shouldn't it be numbers?

Reply:
Some sites aren't too explicit about it but it's mostly used as a string
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Reply:
React accept `number | string` but number seem a better fit IMO

Reply:
Done.